python/src/ops.cpp

```diff
@@ -3645,6 +3645,42 @@ void init_ops(nb::module_& m) {
         Returns:
             array: ``alpha * (a @ b)  + beta * c``
       )pbdoc");
+  m.def(
+      "tile_masked_mm",
```

Comment:
Sorry if this is a nuisance.. but I think the word `block` is more familiar than `tile`. I might prefer the name `block_sparse_mm`  over `tile_masked_mm`. Wdyt?

Reply:
I could change the names to `block_masked_mm`